### ISSUE TITLE
[BUG] App won't launch on ARM Windows 11

### ISSUE THREAD

**Issue by Chester117** [robvdpol/RaceControl#411]
**Describe the bug**
App won't launch on virtualized ARM Windows 11 installed on M1 Macs with Parallels desktop.

**To Reproduce**
after installation, when launching the app, nothing would show up

**Expected behavior**
something to show up

**Desktop (please complete the following information):**
 - Windows 11 ARM on Parallels Desktop(Virtualized)
 - 64 bit

**F1TV account**
- Pro
- US
- Using transparent proxy, so shouldn't be a problem

Just want to know if running on ARM windows is possible or not. Thank you very much!

**Comment by sidewinder94** [comment]
Try this : [RaceControl - ARM64.zip](https://github.com/robvdpol/RaceControl/files/8402441/RaceControl.-.ARM64.zip)

It's an ARM64 build of RaceControl.

There are a few things I suspect won't work at all since they depends on x86/x64 binaries that are not part of the compilation process (the integrated player and MPV being some of them, maybe the download option too).

**Comment by Chester117** [comment]
Thank you very much! I've tried to launch and got this error message
![Screen Shot 2022-04-03 at 16 35 30](https://user-images.githubusercontent.com/40749115/161420172-a8bd9f38-aa66-48c4-ab0a-701822a81a01.jpg)

**Comment by sidewinder94** [comment]
Ok, so yeah, won't work without additional work from @robvdpol. Since FFmpeg is needed as soon as the app starts, and it's one of the binary dependencies.

Sorry can't do much more than that. Especially since I don't have an ARM machine to test with

**Comment by robvdpol** [comment]
ARM is currently not supported. It might happen later though.
